onnxruntime/core/platform/env.h

```diff
@@ -108,8 +104,16 @@ class Env {
 
   virtual int GetNumCpuCores() const = 0;
 
-  // This function doesn't support systems with more than 64 logical processors
-  virtual std::vector<size_t> GetThreadAffinityMasks() const = 0;
+  // Return default threadpool size, and set default affinity vector
+  // Note - affinity.size() does not necessarily amount to the threadpool size
```

Comment:
why? can you document the reasoning?

Reply:
Sure - added detailed comments explaining the design.